root/static/scripts/common/dialogs.js

```diff
@@ -91,7 +91,6 @@ const MB = require('./MB');
             }
 
             var entity = this.options.entity;
-            this.options.title = strings.addANewEntity[entity];
```

Comment:
This is being removed but not substituted by anything - is that intentional?

Reply:
It is substituted at [line no. 454 of `MB/Control/Autocomplete.js`](https://github.com/metabrainz/musicbrainz-server/pull/836/files#diff-a122aa5e611cd820d2a4644b76d96dccR454).